### DISCUSSION TITLE
Can't get started because of an ImportError

### DISCUSSION THREAD

**Discussion by absurdity9** [impira/docquery#69] [General]
Docquery installed correctly. When I run: docquery % docquery scan "What is the invoice number?" https://templates.invoicehome.com/invoice-template-us-neat-750px.png

I get: ImportError: cannot import name 'ModelField' from 'pydantic.fields' (/Users/wxd/miniconda3/lib/python3.10/site-packages/pydantic/fields.py)

Please help.

**Comment by Soumyanetrap** [comment]
Was this issue resolved?
Please let me know How? if it has been resolved.
